types/twine-sugarcube/jquery.d.ts

```diff
@@ -0,0 +1,31 @@
+import { Passage } from "./passage";
+
+declare global {
+    namespace JQuery {
+        interface PassageNavigationEvent<TDelegateTarget = any, TData = any, TCurrentTarget = any, TTarget = any>
+            extends TriggeredEvent<TDelegateTarget, TData, TCurrentTarget, TTarget> {
+            /**
+             * The incoming passage object.
+             */
+            passage: Passage;
+        }
+
+        interface PassageRenderingEvent<TDelegateTarget = any, TData = any, TCurrentTarget = any, TTarget = any>
+            extends PassageNavigationEvent<TDelegateTarget, TData, TCurrentTarget, TTarget> {
+            /**
+             * The element which holds (or will be holding in case of the ':passageinit' event) the redered passage.
+             */
+            content: HTMLElement;
+        }
+
+        interface TypeToTriggeredEventMap<TDelegateTarget, TData, TCurrentTarget, TTarget> {
+            ':passageinit': PassageNavigationEvent<TDelegateTarget, TData, TCurrentTarget, TTarget>;
+			':passagestart': PassageRenderingEvent<TDelegateTarget, TData, TCurrentTarget, TTarget>;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/jquery.d.ts;
  Line: 23;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.